__tests__/app.test.js

```diff
@@ -135,9 +135,41 @@ describe("GET /api/articles", ()=>{
       })
     })
   })
-  test("200: articles sorted", ()=>{
-    
-  })
+  
+    test("200: articles sorted by votes in descending order", ()=>{
+      return request(app)
+      .get('/api/articles?sort_by=votes')
+      .expect(200)
+      .then(({body: {articles}})=>{
+        expect(articles).toBeSortedBy('votes', { descending: true})
+      })
+    })
+    test("200: articles sorted by date in ascending order", ()=>{
+      return request(app)
+      .get('/api/articles?order=ASC')
+      .expect(200)
+      .then(({body: {articles}})=>{
+        expect(articles).toBeSortedBy('created_at', { ascending: true})
```

Comment:
This only passes because the default is descending - false. Please read the documentation before using assertions.